### PR TITLE
add qflist function

### PR DESCRIPTION
**Pull request by chrisbra**
Open location list window when pressing 'Q' (fixes issue #63)

### CODE REVIEW

File: doc/ctrlsf.txt [hunk 1]
```diff
@@ -103,6 +103,9 @@ mode:
   5. If you change your mind later, you can always undo it by pressing
   |u| and saving result buffer again.
 
+Note: The result set will additionally be stored in a |location-list| of the
```
Comment (dyng) [inline doc/ctrlsf.txt]: A bit strange to add this phrase here, moving to the foot of `2.1` looks better.

File: plugin/ctrlsf.vim [hunk 2]
```diff
@@ -134,6 +134,7 @@ let s:default_mapping = {
     \ "next"  : "<C-J>",
     \ "prev"  : "<C-K>",
     \ "pquit" : "q",
+    \ "llist" : "Q",
```
Comment (dyng) [inline plugin/ctrlsf.vim]: Due to the nature of this map does little things, and there is a chance that user has already defined a global mapping for `lopen`, I think it's better to leave this default map empty.

File: autoload/ctrlsf/win.vim [hunk 3]
```diff
@@ -59,6 +59,8 @@ func! ctrlsf#win#OpenMainWindow() abort
     exec 'silent keepalt ' . openpos . winsize . 'split ' . '__CtrlSF__'
 
     call s:InitMainWindow()
+    " add location list
+    call setloclist(winnr(), ctrlsf#db#MatchListQF())
```
Comment (dyng) [inline autoload/ctrlsf/win.vim]: I think it's better to call this function in `ExecSearch` but not here.

**Comment by dyng** [comment]
Looks almost perfect! But in next release branch `v1.3` I have done a refactoring which is related to this feature, please rebase and send p-r to `v1.3`.

**Comment by dyng** [comment]
Another question, why not use quickfix list like ack.vim?

**Comment by chrisbra** [comment]
> Another question, why not use quickfix list like ack.vim?
I don't know how ack.vim uses the quickfix list

**Comment by chrisbra** [comment]
okay, rebased on top of v1.3
Do you need a new p-r?

**Comment by dyng** [comment]
Merged in #67

**Pull request closed without merging**